<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cost_scene</class>
 <widget class="QWidget" name="cost_scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1466</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_18">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Data Collection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_19">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QFrame" name="frame_20">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_49">
            <item>
             <widget class="QFrame" name="frame_88">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_51" stretch="2,5">
               <property name="spacing">
                <number>60</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_89">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QFrame" name="frame_16">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QLabel" name="label_17">
                       <property name="styleSheet">
                        <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
                       </property>
                       <property name="text">
                        <string>Inital Renewable and Energy Storage System Costs</string>
                       </property>
                      </widget>
                     </item>
                     <item alignment="Qt::AlignLeft">
                      <widget class="QPushButton" name="pushButton_3">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../resources.qrc">
                         <normaloff>:/dark_icon/images/dark_icon/help_24dp_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/help_24dp_FILL0_wght200_GRAD0_opsz24.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>24</width>
                         <height>24</height>
                        </size>
                       </property>
                       <property name="flat">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_122">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="cost_pie_layout">
                     <item>
                      <spacer name="verticalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignTop">
                <widget class="QFrame" name="frame_90">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QScrollArea" name="scrollArea">
                    <property name="minimumSize">
                     <size>
                      <width>300</width>
                      <height>360</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="widgetResizable">
                     <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>427</width>
                       <height>360</height>
                      </rect>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_area">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QFrame" name="frame">
                        <property name="frameShape">
                         <enum>QFrame::NoFrame</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_4">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <property name="leftMargin">
                          <number>0</number>
                         </property>
                         <property name="topMargin">
                          <number>0</number>
                         </property>
                         <property name="rightMargin">
                          <number>0</number>
                         </property>
                         <property name="bottomMargin">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLabel" name="label_39">
                           <property name="text">
                            <string>PV Cost ($/MW):</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QFrame" name="frame_44">
                           <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                           </property>
                           <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                           </property>
                           <layout class="QHBoxLayout" name="horizontalLayout_26">
                            <property name="spacing">
                             <number>6</number>
                            </property>
                            <property name="leftMargin">
                             <number>0</number>
                            </property>
                            <property name="topMargin">
                             <number>0</number>
                            </property>
                            <property name="rightMargin">
                             <number>0</number>
                            </property>
                            <property name="bottomMargin">
                             <number>0</number>
                            </property>
                            <item>
                             <widget class="QLineEdit" name="s_cost_input">
                              <property name="text">
                               <string/>
                              </property>
                              <property name="placeholderText">
                               <string>1551273</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QPushButton" name="s_cost_input_store">
                              <property name="maximumSize">
                               <size>
                                <width>0</width>
                                <height>0</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="flat">
                               <bool>true</bool>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                      <item>
                       <widget class="QFrame" name="frame_2">
                        <property name="frameShape">
                         <enum>QFrame::NoFrame</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_5">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <property name="leftMargin">
                          <number>0</number>
                         </property>
                         <property name="topMargin">
                          <number>0</number>
                         </property>
                         <property name="rightMargin">
                          <number>0</number>
                         </property>
                         <property name="bottomMargin">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLabel" name="label_38">
                           <property name="text">
                            <string> Wind Cost ($/MW):</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QFrame" name="frame_43">
                           <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                           </property>
                           <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                           </property>
                           <layout class="QHBoxLayout" name="horizontalLayout_25">
                            <property name="spacing">
                             <number>6</number>
                            </property>
                            <property name="leftMargin">
                             <number>0</number>
                            </property>
                            <property name="topMargin">
                             <number>0</number>
                            </property>
                            <property name="rightMargin">
                             <number>0</number>
                            </property>
                            <property name="bottomMargin">
                             <number>0</number>
                            </property>
                            <item>
                             <widget class="QLineEdit" name="w_cost_input">
                              <property name="text">
                               <string/>
                              </property>
                              <property name="placeholderText">
                               <string>1676095</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <widget class="QPushButton" name="w_cost_input_store">
                              <property name="maximumSize">
                               <size>
                                <width>0</width>
                                <height>0</height>
                               </size>
                              </property>
                              <property name="text">
                               <string/>
                              </property>
                              <property name="flat">
                               <bool>true</bool>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                      <item>
                       <widget class="QFrame" name="frame_3">
                        <property name="frameShape">
                         <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_6">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <property name="leftMargin">
                          <number>0</number>
                         </property>
                         <property name="topMargin">
                          <number>0</number>
                         </property>
                         <property name="rightMargin">
                          <number>0</number>
                         </property>
                         <property name="bottomMargin">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLabel" name="label_37">
                           <property name="text">
                            <string> Power Conversion System Cost ($/MW):</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QFrame" name="frame_21">
                           <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                           </property>
                           <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                           </property>
                           <layout class="QHBoxLayout" name="horizontalLayout_10">
                            <property name="leftMargin">
                             <number>0</number>
                            </property>
                            <property name="topMargin">
                             <number>0</number>
                            </property>
                            <property name="rightMargin">
                             <number>0</number>
                            </property>
                            <property name="bottomMargin">
                             <number>0</number>
                            </property>
                            <item>
                             <widget class="QLineEdit" name="pcs_cost_input">
                              <property name="text">
                               <string/>
                              </property>
                              <property name="placeholderText">
                               <string>347000</string>
                              </property>
                             </widget>
                            </item>
                           </layout>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="cost_scenarios_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
border: none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Please enter inital costs for renewable generation and energy storage systems. Cost scenarios are built with a 2.5% discount rate.&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Cost of photovoltaic power per megawatt ($/MW).&lt;/li&gt;
&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Cost of wind power per megawatt ($/MW).&lt;/li&gt;&lt;/ul&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Cost of a power conversion system per megawatt-hour ($/MW).&lt;/li&gt;
&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Cost of energy storage system per megawatt-hour ($/MWh).&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="hide_cost">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/dark_icon/images/dark_icon/remove_24dp_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/remove_24dp_FILL0_wght200_GRAD0_opsz24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_63">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_30">
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pcs_cost_input_store">
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="es4_cost_input_store">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
